xgo-scripts/packetbeat_before_build.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 
-PB_PATH=/go/src/github.com/elastic/packetbeat
+PB_PATH=/go/src/github.com/elastic/beats/
```

Comment:
Yeah, a bit of a hack there because vendor is no longer in the Packetbeat folder.